buffer/src/main/java/io/netty/buffer/api/internal/LifecycleTracer.java

```diff
@@ -229,11 +278,23 @@ public Trace apply(Stream<StackWalker.StackFrame> frames) {
             throwable.addSuppressed(exception);
         }
 
+        @Override
+        public Throwable getTraceback() {
+            try {
+                return getTraceback(System.nanoTime(), true);
+            } catch (Exception e) {
```

Comment:
why could this throw ?

Reply:
It shouldn't. It was debugging left-over.